components/language_filter/src/main.py

```diff
@@ -0,0 +1,72 @@
+"""A component that filters text based on the language."""
+import logging
+
+import fasttext
+import pandas as pd
+
+from fondant.component import PandasTransformComponent
+from fondant.logger import configure_logging
+
+configure_logging()
```

Comment:
This is outdated. You can rebase on / merge with main and you'll get this for free.